project.clj

```diff
@@ -117,7 +117,10 @@
                        [jsedn "0.4.1"]
                        ;; for twitter bot
                        [twitter "1.7.1"]
-                       [tar-fs "2.0.0"]]}
+                       [tar-fs "2.0.0"]
+                       ;; for solidity sha3 function
+                       [web3-utils "1.0.0-beta.55"]
```

Comment:
LGTM, personally I'd copy the solution from https://github.com/raineorshine/solidity-sha3/blob/master/src/index.js into utils.js, but YMMV